src/lib/opcodes.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  ETHEREUM_EXECUTION_SPECS_COMMIT_ID,
+  ETHEREUM_EXECUTION_SPECS_URL,
+  EVM_OPCODES_URL,
+} from './constants';
+
+export enum OpcodeGroups {
+  Arithmetic = 'arithmetic',
+  Bitwise = 'bitwise',
+  Block = 'block',
+  Comparison = 'comparison',
+  ControlFlow = 'control_flow',
+  Environment = 'environment',
+  Keccak = 'keccak',
+  Log = 'log',
+  Memory = 'memory',
+  Stack = 'stack',
+  Storage = 'storage',
+  System = 'system',
+}
+
+export const ethSpecsOpcodeSrc = (group: OpcodeGroups, line: number): string =>
+  `${ETHEREUM_EXECUTION_SPECS_URL}/blob/${ETHEREUM_EXECUTION_SPECS_COMMIT_ID}/src/ethereum/shanghai/vm/instructions/${group}.py#${line}`;
+
+export const evmCodesOpcodesLink = (opcodeNumber: string): string =>
+  `${EVM_OPCODES_URL}/#${opcodeNumber}`;
```

Comment:
We should change the input type to a number here, then have a method to convert to hex and pad to two digits. Mainly because it's more flexible/clear that way (since you can write `0x52` instead of `52`, where the latter is unclear if it's decimal or hex). We can use something similar to the `formatPrefixByte` method

Reply:
I was too lazy to do it at first, shame on me ^^